docs/pc-widget-unity/available-apis.mdx

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 50
+title: List of Available APIs
+description: "List of available APIs for Helpshift PC widget"
+---
+
+### Helpshift.Initialize
+**Parameters:**
+- path (string): The base path to the Application's streaming assets, e.g., Application.streamingAssetsPath.
+- helpshiftConfig (object): A configuration object for the widget.
+  
+**Description:** Initializes the HelpShift widget and runs it in the background, enabling it to listen for notifications. If notifications are a high priority, this method should be called when the game starts to ensure timely event handling.
+  
+**Example Usage:**
+```
+Helpshift.Initialize(Application.streamingAssetsPath, helpshiftConfig);
+```
+
+### Helpshift.Show
+**Parameters:** None
+
+**Description:**
+
+Displays the HelpShift widget on the screen.
+- This is typically invoked during the on-click action of a clickable element (e.g., a button).
+- `Important:` Ensure that HelpShift.Initialize() has been called earlier in the game flow before invoking this method; otherwise, the widget will not function properly.
+
+**Example Usage:**
+```
+Helpshift.Show()
+```
+
+### HelpShift.Hide
+**Parameters:** None
+
+**Description:**
+
+Hides the HelpShift widget from the screen.
+- `Note:` Clicking the "X" button on the widget also triggers the hide functionality.
```

Comment:
Why code formatting is here? Remove.

Also whole description could be better phrased.